presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -329,121 +330,248 @@ Statistical Aggregate Functions
+        },
 
-        MAP(classification_thresholds(1000, outcome, prediction), classification_recall(1000, outcome, prediction))
+    where :math:`t_j` is the :math:`j`-th smallest threshold,
+    and :math:`y_i`, :math:`x_i`, and :math:`w_i` are the :math:`i`-th
+    entries of ``y``, ``x``, and ``weight``, respectively.
 
-.. function:: regr_intercept(y, x) -> double
+.. function:: classification_precision(buckets, y, x) -> array<double>
 
-    Returns linear regression intercept of input values. ``y`` is the dependent
-    value. ``x`` is the independent value.
+    This function is equivalent to the variant of
+    :func:`classification_precision` that takes a ``weight``, with a per-item weight of ``1``.
 
-.. function:: regr_slope(y, x) -> double
+.. function:: classification_recall(buckets, y, x, weight) -> array<double>
 
-    Returns linear regression slope of input values. ``y`` is the dependent
-    value. ``x`` is the independent value.
+    Computes the recall with up to ``buckets`` number of buckets. Returns
+    an array of recall values.
 
-.. function:: skewness(x) -> double
+    ``y`` should be a boolean outcome value; ``x`` should be predictions, each
+    between 0 and 1; ``weight`` should be non-negative values, indicating the weight of the instance.
 
-    Returns the skewness of all input values.
+    The
+    `recall <https://en.wikipedia.org/wiki/Precision_and_recall#Recall>`_
+    is defined as a sequence whose :math:`j`-th entry is
 
-.. function:: stddev(x) -> double
+    .. math ::
 
-    This is an alias for :func:`stddev_samp`.
+        {
+            \sum_{i \;|\; x_i > t_j \bigwedge y_i = 1} \left[ w_i \right]
+            \over
+            \sum_{i \;|\; y_i = 1} \left[ w_i \right]
+        },
 
-.. function:: stddev_pop(x) -> double
+    where :math:`t_j` is the :math:`j`-th smallest threshold,
+    and :math:`y_i`, :math:`x_i`, and :math:`w_i` are the :math:`i`-th
+    entries of ``y``, ``x``, and ``weight``, respectively.
 
-    Returns the population standard deviation of all input values.
+.. function:: classification_recall(buckets, y, x) -> array<double>
 
-.. function:: stddev_samp(x) -> double
+    This function is equivalent to the variant of
+    :func:`classification_recall` that takes a ``weight``, with a per-item weight of ``1``.
 
-    Returns the sample standard deviation of all input values.
+.. function:: classification_thresholds(buckets, y, x) -> array<double>
 
-.. function:: variance(x) -> double
+    Computes the thresholds with up to ``buckets`` number of buckets. Returns
+    an array of threshold values.
 
-    This is an alias for :func:`var_samp`.
+    ``y`` should be a boolean outcome value; ``x`` should be predictions, each
+    between 0 and 1.
 
-.. function:: var_pop(x) -> double
+    The thresholds are defined as a sequence whose :math:`j`-th entry is
 
-    Returns the population variance of all input values.
+    .. math ::
 
-.. function:: var_samp(x) -> double
+        t_j
```

Comment:
maybe put this one inline, since it's just one variable.  What do you think?

Reply:
@rschlussel You're right. It's better shortened. Thanks!